process/block/postprocess/testIntermediateResult.go

```diff
@@ -0,0 +1,51 @@
+package postprocess
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+	"github.com/ElrondNetwork/elrond-go/hashing"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+// TestIntermediateResProc --
```

Comment:
Please add in the comment that this should only be used for testing purposes not in production code. Example: 
https://github.com/ElrondNetwork/elrond-go/blob/master/process/economics/testEconomicsData.go